pages/tag/[slug].js

```diff
@@ -0,0 +1,135 @@
+import config from "../../config";
+import axios from "axios";
+import Image from "next/image";
+import Link from "next/link";
+import Seo from "../seo";
+import { setPostFields } from "../../utils";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTags } from "@fortawesome/free-solid-svg-icons";
+
+export async function getServerSideProps(context) {
+  var response = null;
+  var posts = [];
+  const slug = context.params.slug;
+  try {
+    response = await axios.get(
+      config.STRAPI_URL + "/v1/tag/" + slug + "?populate=deep&status=published"
```

Comment:
By clicking on a tag, its no showing article from that tag.

Ex: I have one article with the tag `golang`. I clicked on that tag from that article only, and I saw not found message. I think that one article should be shown on that page

Reply:
Have you pull a branch(add-tag-given-and-function)?